x/migration/module/autocli.go

```diff
@@ -60,6 +60,14 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					Skip:           true, // skipped because autoCLI cannot handle signing
 					// TODO_UPNEXT(@bryanchriswhite, #1034): Add morse account claiming CLI, incl. examples (see x/supplier/module/autocli.go).
 				},
+				{
+					RpcMethod:      "ClaimMorseApplication",
+					Use:            "claim-morse-application [morse-src-address] [morse-signature] [stake] [service-config]",
+					Short:          "Send a claim_morse_application tx",
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "morseSrcAddress"}, {ProtoField: "morseSignature"}, {ProtoField: "stake"}, {ProtoField: "serviceConfig"}},
+					Skip:           true, // skipped because autoCLI cannot handle signing
```

Comment:
#PUC `// skipped because autoCLI cannot handle signing`

I'm not clear what this means just by reading the comment

Reply:
```
// skipped because autoCLI cannot handle loading & signing using the Morse key.
```